<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObanalSettings</class>
 <widget class="QWidget" name="ObanalSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Настройка объективного анализа</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="obanalTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tasksTab">
      <attribute name="title">
       <string>Задания объективного анализа</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Приложение:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="appPathLineEdit">
           <property name="placeholderText">
            <string>Путь к исполняемому файлу приложения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectPathButton">
           <property name="toolTip">
            <string>Выбрать приложение</string>
           </property>
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Расписание заданий</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Добавить</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/meteo/icons/tools/plus_black.png</normaloff>:/meteo/icons/tools/plus_black.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Удалить</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/meteo/icons/tools/minus_black.png</normaloff>:/meteo/icons/tools/minus_black.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Редактировать</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/meteo/icons/map/settings.png</normaloff>:/meteo/icons/map/settings.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer6_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>72</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="taskTableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Название</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Кол-во уровней</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Кол-во типов данных</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Кол-во станций</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Следующий запуск</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>72</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="applyButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Сохранить изменения</string>
           </property>
           <property name="text">
            <string>Принять</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Закрыть без сохранения</string>
           </property>
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultsTab">
      <attribute name="title">
       <string>Результаты объективного анализа</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>applyButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>selectPathButton</tabstop>
  <tabstop>appPathLineEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>taskTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
